test/cql-pytest/test_tools.py

```diff
@@ -865,3 +868,83 @@ def test_scrub_validate_mode(scylla_path, scrub_workdir, scrub_schema_file, scru
 
         # Check that validate did not move the bad sstable into qurantine
         assert os.path.exists(scrub_bad_sstable)
+
+
+def _to_cql3_type(t: Type) -> str:
+    # map from Python type to Cassandra type, only a small subset is supported
+    py_to_cql3_type = {int: "Int32Type",
+                       str: "AsciiType",
```

Comment:
UTF8Type (this is what the `text` cql type translates to).

Reply:
okay. changed as suggested.

i wanted to demonstrate an extensive way to build the command line. probably a better way is to use dedicate types as the python driver does to differentiate `UTF8Type` from `AsciiType`.  let's do this when necessary.